docs/source/support/component-statuses.rst

```diff
@@ -0,0 +1,332 @@
+   * - **Node.js Ledger API Bindings**
+     -
+     -
+   * - ``@digital-asset/bindings-js`` :doc:`Node.js library </app-dev/bindings-js>`
+     - Stable
+     - 
+   * - **JavaScript Client Libraries**
+     -
+     -
+   * - ``daml codegen js`` :doc:`CLI and generated code </app-dev/bindings-ts/daml2js>`
+     - Stable
+     -
+   * - ``@daml/types`` :doc:`library and its public API </app-dev/bindings-ts/daml-types>`
+     - Stable
+     -
+   * - ``@daml/ledger`` :doc:`library and its public API </app-dev/bindings-ts/daml-ledger>`
+     - Stable
+     -
+   * - ``@daml/react`` :doc:`library and its public API </app-dev/bindings-ts/daml-react>`
+     - Stable
+     -
+   * - **DAML Libraries**
+     -
+     -
+   * - :doc:`The DAML Standard Library </daml/stdlib/index>`
+     - Stable
+     -
+   * - :doc:`The DAML Script Library </daml-script/api/index>`
+     - Stable
+     -
+   * - :doc:`The DAML Trigger Library </triggers/api/index>`
+     - Stable
+     -
+   
+.. _dev-tools:
+
+Developer Tools
+---------------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **SDK**
+     -
+     -
+   * - Windows SDK (installer_)
+     - Stable
+     -
+   * - :ref:`Mac SDK <mac-linux-sdk>`
+     - Stable
+     -
+   * - :ref:`Linux SDK <mac-linux-sdk>`
+     - Stable
+     -
+   * - :doc:`/tools/assistant` with top level commands
+
+       - ``--help``
+       - ``version``
+       - ``install``
+       - ``uninstall``
+     - Stable
+     -
+   * - ``daml start`` helper command and associated CLI (``daml start --help``)
+     - Stable
+     - 
+   * - ``daml deploy`` :doc:`helper command and associated CLI </deploy/generic_ledger>` (``daml deploy --help``)
+     - Stable
+     -
+   * - Assistant commands to start Runtime Components: ``daml json-api`, ``daml trigger``, ``daml trigger-service``, and ``daml extractor``.
+     - See :ref:`runtime-components`.
+     -
+   * - **DAML Projects**
+     -
+     -
+   * - ``daml.yaml`` project specification
+     - Stable
+     - 
+   * - Assistant commands ``new``, ``create-daml-app``, and ``init``. Note that the templates created by ``daml new`` and ``create-daml-app`` are considered example code, and are not covered by :ref:`semantic versioning <versioning>`.
+     - Stable
+     -
+   * - **DAML Studio**
+     -
+     -
+   * - VSCode Extension
+     - Stable
+     -
+   * - ``daml studio`` assistant command
+     - Stable
+     -
+   * - **Code Generation**
+     -
+     -
+   * - ``daml codegen`` assistant commands
+     - See :ref:`libraries`.
+     -
+   * - **Sandbox Development Ledger**
+     -
+     -
+   * - ``daml sandbox`` assistant command and documented CLI under ``daml sandbox --help``.
+     - Stable
+     -
+   * - DAML Sandbox in Memory (ie without the ``--sql-backend-jdbcurl`` flag)
+     - Stable
+     -
+   * - DAML Sandbox on Postgres (iw with the ``--sql-backend-jdbcurl`` flag)
+     - Stable
```

Comment:
If you declare it now stable, it means we cannot remove sandbox persistence until 12 months after the deprecation, right?

Reply:
That is correct. Though I see our deprecation policy as being on features. Ie I'd see it as legitimate to say "If you want persistence in SDK 2.X, use DAML for Postgres". @cmblitz, do you agree?

Reply:
I agree with @bame-da's statement.  We are not deprecating a feature, only how we refer to it.

Reply:
👍